liblangutil/Scanner.h

```diff
@@ -219,6 +219,12 @@ class Scanner
 	Token skipSingleLineComment();
 	Token skipMultiLineComment();
 
+	/// Tests if current source position is LF or CRLF.
+	bool isLineTerminator() const;
```

Comment:
I would propose to rename this to `atEndOfLine` and return true if `m_char` is either `\r` or `\n`, regardless of what follows.

Reply:
Ok, then we can also remove https://github.com/ethereum/solidity/pull/7174/files#diff-0daa5e48651cfd58baca84f28cbd853bR299, can't we?